packages/ubuntu_wizard/lib/src/widgets/validated_form_field.dart

```diff
@@ -39,9 +40,10 @@ class ValidatedFormField extends StatefulWidget {
   /// The label below the [TextField]
   final String? helperText;
 
-  /// This boolean is forwarded to the [TextField] to decide
-  /// if the digits should be obscured.
-  final bool obscureText;
+  /// If specified, this boolean is forwarded to the [TextField] to decide
+  /// if the digits should be obscured, and a show/hide icon button is shown
```

Comment:
Picking nits: the text field can contain all sorts of characters, not just digits.